nilearn/experimental/surface/_maskers.py

```diff
@@ -175,29 +175,23 @@ class SurfaceLabelsMasker:
 
     Parameters
     ----------
-    labels_img : SurfaceImage object
+    labels_img
         Region definitions, as one image of labels.
 
-    label_names : :obj:`list` of :obj:`str`, default=None
+    label_names
         Full labels corresponding to the labels image.
 
-    Attributes
-    ----------
-    labels_data_ : :obj:`numpy.ndarray`
-
-    labels_ : :obj:`numpy.ndarray`
-
-    label_names_ : :obj:`numpy.ndarray`
     """
 
     # TODO check attribute names after PR 3761 and harmonize with volume labels
     # masker if necessary.
-    labels_img: SurfaceImage
-    label_names: dict[Any, str] | None
 
-    labels_data_: type[np.ndarray]
-    labels_: type[np.ndarray]
-    label_names_: type[np.ndarray]
+    #: Labels array concatenated across both hemispheres
+    labels_data_: np.ndarray
+    #: Unique label values
+    labels_: np.ndarray
+    #: Label names if any, otherwise unique label values
+    label_names_: np.ndarray
```

Comment:
attributes can just have this `#:` syntax with a description above each (which will render under) and no need for an attributes section in the docstring. See e.g.
![image](https://github.com/nilearn/nilearn/assets/63292494/e668df22-0f75-4c49-94f0-9f7e978493f3)

Reply:
The pro is that the documentation is closer to where the attributes are declared in the code but this would mean rewriting a lot of doc, no?